### DISCUSSION TITLE
Welcome to Royal Madness Discussions!

### DISCUSSION THREAD

**Discussion by hmbanan666** [chat-game/space#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andreymeretsky** [comment]
Hi, @hmbanan666!
What game mechanics are you wondering to implement firstly?

**Reply by hmbanan666** [reply]
Hi! Working on characters. They are running on screen while Twitch stream is on
